modules/hal_bouffalolab/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ zephyr_library_compile_definitions(
   BFLB_USE_CUSTOM_LD_SECTIONS
 )
 
-set(bflb_soc         ${CONFIG_SOC_SUB_SERIES})
+set(bflb_soc         ${CONFIG_SOC_SERIES}02)
```

Comment:
why are you changing something you just added in another commit? Fix it in the original commit